vector/src/main/java/im/vector/app/features/notifications/NotifiableEventResolver.kt

```diff
@@ -153,7 +153,8 @@ class NotifiableEventResolver @Inject constructor(
             event.attemptToDecryptIfNeeded(session)
             // only convert encrypted messages to NotifiableMessageEvents
             when (event.root.getClearType()) {
-                EventType.MESSAGE -> {
+                EventType.MESSAGE,
+                EventType.POLL_START -> {
```

Comment:
will be replaced by `in EventType.POLL_START` once #5345 will be merged (there is just a minor fix to do there)